website/snippets/_adapters-trusted.md

```diff
@@ -94,8 +94,8 @@
 
 <Card
     title="Teradata"
-    body="<ul><li><a href='/docs/core/connect-data-platform/teradata-setup'>Install with dbt Core  </a> </li> </ul><br /><br /> <a href=https://badge.fury.io/py/dbt-teradata><img src=https://badge.fury.io/py/dbt-teradata.svg/></a>"
-    pills='["dbt Core"]'
+    body="<ul><li><a href='/docs/cloud/connect-data-platform/connect-teradata'>Set up in dbt Cloud  </a> <br /></li> <li><a href='/docs/core/connect-data-platform/teradata-setup'>Install with dbt Core  </a> </li> </ul><br /><br /> <a href=https://badge.fury.io/py/dbt-teradata><img src=https://badge.fury.io/py/dbt-teradata.svg/></a>"
+    pills='["dbt Cloud", "dbt Core"]'
```

Comment:
sugg fixing the PY and pills alignment. it's shifted down compared to the other cards in the same row:

 
![Screenshot 2024-10-07 at 12 06 30 PM](https://github.com/user-attachments/assets/26ce98ae-5cc8-4d07-95ab-d672b53ac220)